### PR TITLE
feat(dotenv) add dotENV icon

### LINKED ISSUE #1563: [Icon Request] Support DotEnv
- [x] I'm sure this issue is not a _duplicate_.
- [x] I'm submitting an icon request:
  - Type: `language id`
  - Icon Name: `config`
  - Sample original Icon: Same as `config`
  - Language ids: `dotenv`

- Language Extensions: https://marketplace.visualstudio.com/items?itemName=mikestead.dotenv

### PR DESCRIPTION
**Pull request by voidei**
_**Fixes #1563 #2278**_

Adds the official dotENV icon for all [supported](https://github.com/dotenv-org/dotenv-vscode/blob/master/package.json#L78) .env files.

Trying this again 👍

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

_Icon Source:_

- [normal icon](https://github.com/dotenv-org/dotenv-vscode/blob/master/media/sidebar-icon.svg)
- [light icon](https://github.com/dotenv-org/media-kit/blob/master/logos/dotenv-monochrome.svg)

dotenv
dotenv

### CODE REVIEW

**Comment by robertohuertasm** [comment]
Regarding the icon, I would only use the yellow one, which is the official one and it works well for both themes. Aside from that, I see that the `ENV` text is black in the original one. I would keep that, too.

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -1436,6 +1432,68 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'dotenv',
+      extensions: [
+        '.env-sample',
+        '.env',
+        '.fart',
```
[the remark this answers is not in the document]
Comment (voidei) [reply src/iconsManifest/supportedExtensions.ts:1440]: File extensions were taken from the official [VSCode dotENV extension](https://github.com/dotenv-org/dotenv-vscode), can find the supported extensions listed [here](https://github.com/dotenv-org/dotenv-vscode/blob/master/package.json#L78)

**Review by voidei — COMMENTED** [review]
Accidentally resolved this too early, but I fixed up the extensions for .env

**Comment by voidei** [comment]
Just a quick update on the light theme version, which was also taken from the official dotENV monochrome icon [here](https://github.com/dotenv-org/media-kit/blob/master/logos/dotenv-monochrome.svg) - I've attached two screenshots showing the differences between it and the yellow in a light theme I have installed for VSCode, as the yellow really doesn't work well due to how bright it is.
dark_dotenv
light_dotenv

**Comment by robertohuertasm** [comment]
The yellow one doesn't work well because the letters are transparent. Check how it looks in the official [dotenv page](https://www.dotenv.org/)

image

**Comment by voidei** [comment]
@robertohuertasm Ah I see - my mistake then, just fixed those up. Should be good to go :+1:

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

**Pull request merged**